src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -74,18 +74,31 @@ def run_check(self, ctx: AnalyzeContext) -> CheckResultData:
         CheckResultData
             The result of the check.
         """
-        available = (
-            ctx.dynamic_data["provenance_info"]
-            and ctx.dynamic_data["provenance_info"].provenance_payload
-            and not ctx.dynamic_data["is_inferred_prov"]
-        )
+        provenance_info = None
+        inferred = False
+        if ctx.dynamic_data["provenance_info"]:
+            provenance_info = ctx.dynamic_data["provenance_info"]
+            inferred = ctx.dynamic_data["is_inferred_prov"]
+
+        if not provenance_info or not provenance_info.provenance_payload or inferred:
+            return CheckResultData(
+                result_tables=[
+                    ProvenanceAvailableFacts(
```

Comment:
Looks like we are not storing any useful info, so I think we can skip this? Or am I missing something here? :thinking: